internal/bitcoin/bitcoin.go

```diff
@@ -33,9 +33,9 @@ type BitcoinDto struct {
 	sharedAPI.Resources
 }
 
-type BitcoinListDto []BitcoinDto
+type BitcoinListDto []*BitcoinDto
```

Comment:
Use a slice of structs instead of a slice of pointer to structs.
Do the same for all other protocols